pom.xml

```diff
@@ -126,6 +127,16 @@
       <artifactId>mailer</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>com.thoughtworks.xstream</groupId>
+      <artifactId>xstream</artifactId>
+      <version>1.4.19</version>
+    </dependency>
```

Comment:
The xstream API is already provided by Jenkins core without requiring the inclusion of a dependency.  This needs to be removed.  It causes several additional transitive dependencies to be included that should not be in the git plugin.
```suggestion
```